docs/sphinx/reference-libobs-callback.rst

```diff
@@ -286,3 +286,15 @@ direct access to declarations or callback pointers.
    :param handler: Procedure handler object
    :param name:    Name of procedure to call
    :param params:  Calldata structure to pass to the procedure
+   
+
+---------------------
+
+.. function:: bool proc_handler_exists(proc_handler_t *handler, const char *name, calldata_t *params)
```

Comment:
Unexpected parameter `params` in documentation.  The actual function `proc_handler_exists` only takes two parameters:  `proc_handler_t *handler` and `const char *name`.

Reply:
Stupid mistake. Fixed.